### PR TITLE
Clarify supported versions

### PR DESCRIPTION
**Pull request by friism**
cc @moscowrage @carlfischer1

### CODE REVIEW

File: engine/installation/windows/docker-ee.md [hunk 1]
```diff
@@ -10,9 +10,9 @@ redirect_from:
 {% capture download_url %}https://download.docker.com/components/engine/windows-server/{{ site.docker_ee_version }}/{{ filename }}{% endcapture %}
 
 
-Docker Enterprise Edition for Windows Server 2016 (*Docker EE*) enables native
-Docker containers on Windows Server 2016. The Docker EE installation package
-includes everything you need to run Docker on Windows Server 2016.
+Docker Enterprise Edition for Windows Server (*Docker EE*) enables native
+Docker containers on Windows Server. Windows Server 2016 and later versions are supported (currently Windows Server 2016 and Windows Server 1709). The Docker EE installation package
```
Comment (mistyhacks) [inline engine/installation/windows/docker-ee.md]: This will be a pain to maintain...

Comment (friism) [reply engine/installation/windows/docker-ee.md]: @mistyhacks want it to just say windows server 2016 and later?

Comment (carlfischer1) [reply engine/installation/windows/docker-ee.md]: +1

Comment (mistyhacks) [reply engine/installation/windows/docker-ee.md]: Yes please!

@carlfischer1 any relation to the [music publisher](https://www.carlfischer.com/) ?? 🎶

**Comment by friism** [comment]
@mistyhacks ptal

**Pull request merged**